src/python/generate_singa_wrapper.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+!/usr/bin/env bash
```

Comment:
I have already moved the functions in the cmake scripts.
The bash file is only for standalone test.